onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorLayerNormalization.cpp

```diff
@@ -6,6 +6,7 @@
 namespace Dml
 {
 
+template <bool simplified>
```

Comment:
>simplified [](http://example.com/codeflow?start=15&length=10)

nit: should template parameters be prefixed with T as in TSimplified?

Reply:
T is only for types right? In this case it acts more like a variable.